frontend/src/app/api/tests/[testId]/results/route.ts

```
@@ -138,6 +138,9 @@ export async function GET(
       );
     }
 
+    // Round all scores to integers
+    categoryScores.forEach(cs => cs.score = Math.round(cs.score));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)